yt/go/yt/internal/call.go

```diff
@@ -56,6 +58,11 @@ func (res *CallResult) decode(value any) (err error) {
 	return
 }
 
+func (res *CallResult) decodeJSON(value any) (err error) {
+	err = json.Unmarshal(res.YSONValue, value)
+	return
+}
```

Comment:
Could you provide some context (or maybe in the code comment also) 
1) why CallResult require JSON decoder now and didn't need it before? All other methods are returning YSON?
2)Am I understanding correctly that something called YSONValue contains JSON?

Reply:
1. Even if we specify the output format is specified in header as YSON the whoami handler returns the JSON value.
2. YSONValue contains the byte representation of response body, it's inly named YSON

I've added couple more comments here and there.